workflow/Snakefile

```diff
@@ -15,10 +15,13 @@ module dna_seq_varlociraptor:
 use rule * from dna_seq_varlociraptor
 
 
-if "normal" not in dna_seq_varlociraptor.samples["alias"]:
-    del dna_seq_varlociraptor.config["calling"]["fdr-control"]["events"][
-        "loh_or_amplification"
-    ]
+if "normal" not in dna_seq_varlociraptor.samples["alias"].unique():
+    list_keys = list(
+        dna_seq_varlociraptor.config["calling"]["fdr-control"]["events"].keys()
+    )
+    for k in list_keys:
+        if k.startswith("loh"):
+            dna_seq_varlociraptor.config["calling"]["fdr-control"]["events"].pop(k)
```

Comment:
I think this needs a little comment. Why again are we deleting something here?

Reply:
Because we only consider LOH in case there is a tumor and normal sample.
It would probably be nicer to add the `loh` events only to the config in case the normal sample is present.
But as we can not check when rendering the config we have to remove the events afterwards.

Reply:
Ah, I remember, thanks!